### PR TITLE
Initial source

### PR DESCRIPTION
**Pull request by fatroom**

### CODE REVIEW

**Comment by fatroom** [comment]
👍

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,98 @@
+
+# Created by https://www.gitignore.io/api/intellij+iml,maven,osx
```
Comment (twz123) [inline .gitignore]: If you list everything here, it's unfair not to include Eclipse ;)

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.zalando</groupId>
+    <artifactId>gson-money</artifactId>
```
Comment (twz123) [inline pom.xml]: Shouldn't this match the repo name?

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    <version>0.0.1-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>Gson Money</name>
+    <url>http://maven.apache.org</url>
```
Comment (twz123) [inline pom.xml]: Should be the GitHub URL, I'd say. Just look at the [Undertaking POM][ut-pom], lines 13 to 33.

[ut-pom]: https://github.com/zalando-incubator/undertaking/blob/0.0.5/pom.xml#L13

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+            <version>2.7</version>
+        </dependency>
+        <!-- Money Support-->
+        <dependency>
+            <groupId>javax.money</groupId>
+            <artifactId>money-api</artifactId>
+            <version>1.0.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.javamoney</groupId>
```
Comment (twz123) [inline pom.xml]: For which reason do we need an actual implementation as a runtime dependency?

Comment (fatroom) [inline pom.xml]: Should be available during the tests. Will change to scope test.

**Comment by twz123** [comment]
:+1:

**Comment by fatroom** [comment]
👍

**Pull request merged**